scm-webapp/src/test/java/sonia/scm/security/AnonymousRealmTest.java

```diff
@@ -56,7 +59,6 @@
   @Mock
   private UserDAO userDAO;
 
-  @InjectMocks
```

Comment:
Do we need no longer mock injection?

Reply:
No, since the realm is initialized in the beforeEach anyway.